### PR TITLE
[exmaple] add bert and albert

### PR DESCRIPTION
**Pull request by feifeibear**
## 📌 Checklist before creating the PR

- [x] I have created an issue for this PR for traceability
- [ ] The title follows the standard format: `[doc/gemini/tensor/...]: A concise description`
- [ ] I have added relevant tags if possible for us to better distinguish different PRs

## 🚨 Issue number

> Link this PR to your issue with words like fixed to automatically close the linked issue upon merge
>
> e.g. `fixed #1234`, `closed #1234`, `resolved #1234`

## 📝 What does this PR do?

> Summarize your work here.
> if you have any plots/diagrams/screenshots/tables, please attach them here.

## 💥 Checklist before requesting a review

- [x] I have linked my PR to an issue ([instruction](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue))
- [x] My issue clearly describes the problem/feature/proposal, with diagrams/charts/table/code if possible
- [x] I have performed a self-review of my code
- [ ] I have added thorough tests.
- [ ] I have added docstrings for all the functions/methods I implemented

## ⭐️ Do you enjoy contributing to Colossal-AI?

- [x] 🌝 Yes, I do.
- [ ] 🌚 No, I don't.

Tell us more if you don't enjoy contributing to Colossal-AI.

### CODE REVIEW

File: examples/language/bert/train_bert_demo.py [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+from colossalai.nn.parallel import zero_model_wrapper, zero_optim_wrapper
+from colossalai.tensor import ColoParameter, ComputePattern, ComputeSpec, ProcessGroup, ReplicaSpec, ShardSpec
+from colossalai.utils import get_current_device
+from colossalai.utils.model.colo_init_context import ColoInitContext
+
+CAI_VERSION = colossalai.__version__
+
+
+def get_tflops(model_numel, batch_size, seq_len, step_time):
+    return model_numel * batch_size * seq_len * 8 / 1e12 / (step_time + 1e-12)
```
Comment (1SAA) [inline examples/language/bert/train_bert_demo.py:24]: I think we can move these utility functions into another file or move them to Colossalai if they are used in both GPT models and BERT models.

Comment (feifeibear) [reply examples/language/bert/train_bert_demo.py:24]: Yes. But we want the user to run their examples in the current directory. Using common utils needs dependencies on its parent directory. You can fix them in another PR.

Comment (1SAA) [reply examples/language/bert/train_bert_demo.py:24]: We can just put these functions into Colossalai if necessary.

Comment (feifeibear) [reply examples/language/bert/train_bert_demo.py:24]: OK, and just put them here is OK. Each example is an independent project.

**Review by 1SAA — APPROVED** [review]

**Pull request merged**